packages/wifi_iot/android/src/main/java/com/alternadom/wifiiot/WifiIotPlugin.java

```diff
@@ -348,12 +351,28 @@ private void getWiFiAPSSID(Result poResult) {
         return;
       }
 
-      poResult.error("Exception", "SSID not found", null);
+      poResult.error("Exception [getWiFiAPSSID]", "SSID not found", null);
     } else {
-      poResult.error(
-          "Exception [getWiFiAPSSID]",
-          "Getting SSID name is not supported on API level >= 26",
-          null);
+      if (apReservation != null) {
+        String ssid;
+        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.R) {
+          WifiConfiguration wifiConfiguration = apReservation.getWifiConfiguration();
+          if (wifiConfiguration != null) {
+            ssid = wifiConfiguration.SSID;
+          } else {
+            poResult.error(
+                "Exception [getWiFiAPSSID]",
+                "Security type is not WifiConfiguration.KeyMgmt.None or WifiConfiguration.KeyMgmt.WPA2_PSK",
+                null);
+          }
+        } else {
+          SoftApConfiguration softApConfiguration = apReservation.getSoftApConfiguration();
+          ssid = softApConfiguration.getSsid();
+        }
+        poResult.success(ssid);
```

Comment:
Wrongly placed, in case of `wifiConfiguration == null`, it will call both `poResult.error` and `poResult.success`

Reply:
Done!